### DISCUSSION TITLE
Why did you name your folders after dinner items?

### DISCUSSION THREAD

**Discussion by samarmohan** [benawad/dogehouse#249] [General]
This just struck me, and I am confused.

**Comment by sebasptsch** [comment]
Because food is modular. You could put all these foods together and you'll get a **unique** end product.

**Reply by Silverman42** [reply]
This seems valid

**Comment by DGKSK8LIFE** [comment]
for memeable value @samarmohan

**Comment by samarmohan** [comment]
That’s valid
